library/src/G4CMPPhononBoundaryProcess.cc

```diff
@@ -554,10 +569,8 @@ AdjustToClosestSurfacePoint(const G4VSolid* solid,
   if (isIn == kSurface) return;
 
   // Angles to be adjusted in place by OptimizeSurfaceAdjustAngle
-  // Best initial conditions will be the surface normal
-  G4ThreeVector surfNorm = solid->SurfaceNormal(stepLocalPos);
-  G4double bestTheta = surfNorm.theta();
-  G4double bestPhi = surfNorm.phi();
+  G4double bestTheta = pi / 2;
+  G4double bestPhi = 0;
```

Comment:
Why not use the surface normal (or its reflection) as the initial guess?

Reply:
Oh!  It's a gimbal lock situation.  I understand.  It might be worth putting in a one-line comment mentioning why you have it start at the theta pole.